punx/tests/test_issue130.py

```diff
@@ -0,0 +1,58 @@
+"""
+Issue 130: BUG: tree cannot parse some NeXus examples
+"""
+
+import h5py
+import os
```

Comment:
```suggestion
import os

import h5py
```
Usually imports are grouped by system, other, local.